test/unit/ngsiv2/provisioning/singleConfigurationMode-test.js

```diff
@@ -0,0 +1,333 @@
+/*
+ * Copyright 2016 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of fiware-iotagent-lib
+ *
+ * fiware-iotagent-lib is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * fiware-iotagent-lib is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with fiware-iotagent-lib.
+ * If not, seehttp://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License
+ * please contact with::[contacto@tid.es]
+ *
+ * Modified by: Daniel Calvo - ATOS Research & Innovation
```

Comment:
shouldn't this be Author: Daniel Calvo - ATOS Research & Innovation

Reply:
This test is just a small adaptation of https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/unit/provisioning/singleConfigurationMode-test.js for NGSIv2 features. To be honest, I am just a contributor bit not the `original` author.

Reply:
NTC